src/components/password/password.test.js

```diff
@@ -0,0 +1,190 @@
+import React from "react";
```

Comment:
Might not be for this PR or if one of the QA's is kind enough, this file probably needs an `checkAccessibility()` test(s).